docker/.env.example

```diff
@@ -2,6 +2,6 @@ ASPNETCORE_ENVIRONMENT=""
 AzureAd__ClientID=""
 AzureAd__ClientSecret=""
 AzureAd__TenantID=""
-AzureAd__RedirectUri=""
```

Comment:
Why has this been removed?

Reply:
Because we don't actually override it so when we make a .env file from this example we delete this line. It's kinda a mistake for it to be here so I sneakily tidied it